stdpopsim/models.py

```diff
@@ -242,40 +204,53 @@ def debug(self, out_file=sys.stdout):
             demographic_events=self.demographic_events)
         dd.print_history(out_file)
 
-    # TODO deprecate/remove this, as the recommended interface is now 'run'.
-    def asdict(self):
-        return {
-            "population_configurations": self.population_configurations,
-            "migration_matrix": self.migration_matrix,
-            "demographic_events": self.demographic_events}
+    def get_samples(self, *args):
```

Comment:
using the `*args` syntax here is a bit cryptic for the user. can the doc string be a bit more descriptive?

Reply:
Agreed, this is cryptic. I've added a TODO for now.